vectordatasource/transform.py

```diff
@@ -9624,3 +9624,25 @@ def capital_alternate_viewpoint(shape, props, fid, zoom):
                 props['region_capital:' + viewpoint] = False
 
     return shape, props, fid
+
+
+_ADMIN_LEVEL_TO_KIND = {'2': 'country', '4': 'region', '6': 'county', '8': 'locality'}
+
+
+def admin_level_alternate_viewpoint(shape, props, fid, zoom):
+    """
+    turns e.g. admin_level:XX=4 into kind:XX=region
+    """
+    admin_viewpoint_prefix = 'admin_level:'
+    tags = props['tags']
+
+    for k in tags.keys():
+        if k.startswith(admin_viewpoint_prefix):
+            viewpoint = k[len(admin_viewpoint_prefix):].upper()
```

Comment:
This should instead be `lower()` as Tilezen's convention is all lowercase property names (so `kind:us` not `kind:US`).